### ISSUE TITLE
app-template-svelte: Dependency svelte/internal not found!

### ISSUE THREAD

**Issue by dominikg** [pikapkg/create-snowpack-app#75]
a fresh start with app-template svelte fails to start/build

setup:
```
npx create-snowpack-app snowpack-svelte-app --template @snowpack/app-template-svelte
cd snowpack-svelte-app
```

`npm start`
```
Snowpack

  http://localhost:8080 > http://192.168.2.119:8080
  Server started in 5ms.

  mount:web_modules.......[DONE] mount $WEB_MODULES --to /web_modules
  mount:public............[DONE] mount public --to /
  mount:src...............[DONE] mount src --to /_dist_
  build:js,jsx,ts,tsx.....[READY] (default) esbuild
  build:svelte............[READY] (plugin) @snowpack/plugin-svelte
```
* browser opens on localhost:8080
* no visible content on page 
* 404 response from http://localhost:8080/web_modules/svelte/internal.js

`npm run build`
```
Snowpack

  ./build Building your application...

  mount:web_modules.......[DONE] mount $WEB_MODULES --to /web_modules
  mount:public............[DONE] mount public --to /
  mount:src...............[DONE] mount src --to /_dist_
  build:js,jsx,ts,tsx.....[DONE] (default) esbuild
  build:svelte............[RUNNING] (plugin) @snowpack/plugin-svelte
  bundle:*................[SKIP] 

▼ Snowpack

  Dependency svelte/internal not found!
```

versions:
os: ubuntu 18.04
node: 12.17.0
npm: 6.14.5
create-snowpack-app: 1.0.0

**Comment by Evertt** [comment]
Wow, I literally just wanted to open the same issue!

**Comment by FredKSchott** [comment]
Thanks for filing! Looks like we made a bad config change as we got the 2.0.0 release ready.

Fixed in https://github.com/pikapkg/create-snowpack-app/commit/eb4a9172056d7cd021e0a55e1959efd2ae01d15f, and released as `v1.0.1`. Update all of your deps to get the fix automatically.
